src/xopr/cf_units.py

```diff
@@ -1,23 +1,83 @@
-# TODO: This is entirely AI generated and unchecked.
-# I just wanted something to quickly demonstrate what units might looks like.
-# Should be carefully reviewed.
+"""
+CF-compliant metadata and units for polar radar datasets.
+
+This module provides utilities for applying Climate and Forecast (CF) metadata
+conventions to polar radar echogram datasets. The CF conventions ensure that
+radar data includes standardized attributes for coordinates, data variables,
+and global metadata, making the datasets more interoperable and self-describing.
+
+The primary function applies CF-1.8 compliant attributes to xarray Datasets,
+including:
+- Standard names and units for coordinates (time, two-way travel time)
+- Physical units and descriptions for data variables (radar power, GPS position, etc.)
+- Global attributes for dataset provenance and spatial/temporal coverage
+
+Notes
+-----
+This is a work in progress and the data structures are definitely not fully CF-compliant yet.
+
+@private
```

Comment:
... I feel like we don't want `@private` here? Adding this will exclude the whole module from the API docs generation. Did you mean to exclude it because it's a work in progress, or is this accidental?